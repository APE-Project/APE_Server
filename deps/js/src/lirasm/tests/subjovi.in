	ptr = allocp 8

	a = immi -2147483647
	b = immi 1
	c = subjovi a b ovf
	sti c ptr 0

	j done

ovf:	i = immi 12345678
	sti i ptr 0

done:	res = ldi ptr 0
	reti res

