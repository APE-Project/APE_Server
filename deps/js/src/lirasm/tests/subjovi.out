Output is: -2147483648
